src/transformers/onnx/config.py

```diff
@@ -178,6 +196,21 @@ def atol_for_validation(self) -> float:
         """
         return 1e-5
 
+    @property
+    def is_torch_support_available(self) -> bool:
+        """
+        The minimum PyTorch version required to export the model.
+
+        Returns:
+            Bool
```

Comment:
Here I'm following the conventions on the pre-existing doc strings. Will tidy them up in a separate PR

Reply:
I don't see what's preventing us from writing a proper docstring here now.

Reply:
Done!